topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -155,6 +156,8 @@ private Value(String key, Type<? extends T> type, boolean required) {
     public static final Value<String> TLS_KEYSTORE_LOCATION = new Value<>(TC_TLS_KEYSTORE_LOCATION, STRING, "");
     /** The password for keystore with private key and certificate for client authentication against Kafka broker */
     public static final Value<String> TLS_KEYSTORE_PASSWORD = new Value<>(TC_TLS_KEYSTORE_PASSWORD, STRING, "");
+    /** The endpoint identification algorithm used by clients to validate server host name. The default value is https. Clients including client connections created by the broker for inter-broker communication verify that the broker host name matches the host name in the broker’s certificate. Disable server host name verification by setting to an empty string.**/
+	public static final Value<String> TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG = new Value<>(TC_TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG, STRING, "HTTPS");
```

Comment:
Same as above - do we need the the `_CONFIG` at the end? It is not used on the other fields.

Reply:
removed